static/styles/left-sidebar.scss

```diff
@@ -202,31 +202,34 @@ li.show-more-topics {
 ul.filters {
     list-style-type: none;
     margin-left: 0px;
-}
 
-ul.filters a {
-    color: inherit;
-}
+    a {
+        color: inherit;
+    }
 
-ul.filters hr {
-    margin-top: 10px;
-    margin-bottom: 10px;
-}
+    hr {
+        margin-top: 10px;
+        margin-bottom: 10px;
+    }
 
-ul.filters li.muted_topic,
-ul.filters li.out_of_home_view {
-    opacity: 0.25;
-}
+    li.muted_topic {
+        opacity: 0.25;
+    }
 
-ul.filters li.out_of_home_view:hover {
-    opacity: 0.6;
-}
+    li.out_of_home_view {
+        @extend li.muted_topic;
```

Comment:
I'd again say, preserve the commas. I believe they are there for a reason. You usually extend from an abstract class and not a functional class.

Here this also causes issues since `li.muted_topic` is again nested inside this, which also makes the code harder to look at. These kind of changes require a lot of proof that they work, and instead we could just go for a simple solution.